libraries/cyclestreets-core/src/main/java/net/cyclestreets/api/Maps.kt

```diff
@@ -0,0 +1,44 @@
+package net.cyclestreets.api
+
+import android.os.AsyncTask
+
+class Maps(
+        private val packs: Collection<Map>
+): Iterable<Map> {
```

Comment:
I think this would read easier as a one-liner rather than these 3 lines, but maybe that's just me.